gensim/matutils.py

```diff
@@ -479,6 +479,38 @@ def isbow(vec):
         return False
     return True
 
+def symmetric_kl(distrib_p, distrib_q):
+    return numpy.sum([stats.entropy(distrib_p, distrib_q), stats.entropy(distrib_p, distrib_q)])
```

Comment:
Why `numpy.sum` instead of `+`?

Reply:
How is it a matrix operation? It seems to add two numbers.

Reply:
The ``distrib_p`` and ``distrib_q`` are two matrices from which the Symmetric KL divergence is calculated. The exact formula(which is adapted from the R package) is ``divergence <- sum(cm1*log(cm1/cm2)) + sum(cm2*log(cm2/cm1))`` which is similar to what this function does.

Reply:
That's a non-standard use of KL (which works over a probability distribution = vector). 
Deserves a comment & proper docstring.

What is actually `distrib_p` and `distrib_q`? Why are they matrices?